### DISCUSSION TITLE
When I run `qsv --update` or `qsv --updatenow` nothing happens.

### DISCUSSION THREAD

**Discussion by jqnatividad** [dathere/qsv#769] [FAQ]
This is because GitHub is rate-limiting qsv usage of the GitHub API to check for new releases.

If you turn on logging - i.e. `export QSV_LOG_LEVEL=debug` you'll see if the GitHub API is rate-limiting qsv.

Just wait a few minutes and try again.

Already, GitHub is very generous in hosting the distribution of binaries for open source projects.  But should qsv get more popular and we regularly get rate-limited, we may also host the binaries on S3 so we don't have to worry about GitHub rate-limiting.

**Comment by aborruso** [comment]
I had no idea it was that easy, thank you

**Comment by jqnatividad** [comment]
Hi @aborruso ,
I already made a change to master so it won't fail `--update` silently and just log the error.  It will also show the error on stderr.

https://github.com/jqnatividad/qsv/commit/711e17ac46f4e03d72ced6e49e1bf5dc7d1d5880

And frankly, it's a nice problem to have, knowing that qsv is being used enough in the wild that we hit rate limits... 😄
